src/theme/components/footer.ts

```diff
@@ -0,0 +1,132 @@
+import { displayScreenreaderOnly, screenreaderOnly } from "./globalMixins";
+
+export const footerGray = "#54514A";
```

Comment:
Great.  This gray sits almost half way between `ui.gray.dark` and `ui.gray.x-dark`.  If it was closer to one or the other, I would push to make the jump to our gray, but not when it's this far off.  Too bad.